index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/style.css">
+  <title>Do it!</title>
```

Comment:
Please make the commit smaller next time.

Reply:
Also, you forgot to add the bootstrap.css

Reply:
Noted and thanks, LR!